crates/resolver/src/lookup.rs

```diff
@@ -84,12 +84,13 @@ impl Lookup {
         &self.query
     }
 
-    /// Returns a borrowed iterator of the returned IPs
+    /// Returns an iterator over the response records that match the queried record type.
     pub fn iter(&self) -> LookupIter<'_> {
         LookupIter(self.records.iter())
     }
 
-    /// Returns a borrowed iterator of the returned IPs
+    /// Returns an iterator over all records that were returned during the query, this can include
```

Comment:
Style nit: for this and the next one, I generally try to keep a summary that fits on one line for docstrings, then an empty line, then additional details. In this case, I think it would be nice to just do "May include additional record types ..".

Reply:
updated.